README.md

```diff
@@ -3,22 +3,101 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-8-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
 EOS Costa Rica Developers Documentation Site.
 
 In this site you will find the resources collected in the day-to-day life of EOS Costa Rica collaborators. Feel free to collaborate, comment and suggest changes / ideas to improve the site.
 
+<img
+  alt="Guide front page"
+  src={useBaseUrl('img/guide-page.png')}
+/>;
 
-## Contributing
+## About the Project
 
-We use a Kanban-style board. That's were we prioritize the work. [Go to Project Board](https://github.com/eoscostarica/guide.eoscostarica.io/projects/1).
+This project recollect all new knowledge gotten by our workers to help and share information.
+
+## Table of Contents
+
+1. About The Project
+1. Content Table
+1. Project Purpose
+1. Version
+1. Tech Stack
+1. Development Environment
+1. File Structure
+1. License
+1. Contributing
+1. Contributors
+1. About Company
+
+## Project Purpose
+
+We have created this project to improve our workflow which allows the old and new member have a guideline to follow to help them feel as comfortable as possible following the programming practices, recommendations, programming bases and others.
+
+## Version
+:::note modify
+we are on version 0.2.0
+:::
+
+## Tech Stack
+- React.js - React 16✨, React Router 5
+- Docusaurus - Build optimized websites quickly, focus on your content
+- Material-ui/core - React components for faster and easier web development universal-authenticator-library - A library for allowing apps to easily use different auth providers
+- Lint - ESlint
+- Styles - Material-UI Theme (customizable)
+
+
+## Development Environment
+
+[evodex.io](https://evodex.io/exchange) is running on the EOS MainNet and is built from the master branch, our production branch.
 
-Contributing Guidelines  https://guide.eoscostarica.io/docs/open-source-guidelines .
+[jungle.evodex.io](https://jungle.evodex.io/) is running on the Jungle 3 TestNet and is built from the develop branch used for development, integration, and testing new features.
```

Comment:
@xavier506 I leave it as a reference of the Evodex project, what could I add here to the guide project

Reply:
remove this section we don't have a development environment for this project